bank_statement_parse_camt/README.rst

```diff
@@ -0,0 +1,3 @@
+Module to import SEPA CAMT.053 Format bank statement files.
```

Comment:
README.rst must be formatted as https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst